source/shared/cpp/ObjectModel/BaseActionElement.cpp

```diff
@@ -134,6 +145,11 @@ Json::Value BaseActionElement::SerializeToJsonValue() const
         root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::IsEnabled)] = m_isEnabled;
     }
 
+    if (m_role != ActionRole::Button)
```

Comment:
why this check here?:)

Reply:
Button is the default so there's no need to update it

Reply:
gotcha